Cargo.toml

```
@@ -79,8 +79,10 @@ members = [
     "crates/rpc/rpc-builder/",
     "crates/rpc/rpc-engine-api/",
     "crates/rpc/rpc-eth-api/",
+    "crates/rpc/rpc-eth-server-types/",
```

let's just do eth or eth-types, perhaps the latter?
```suggestion
    "crates/rpc/rpc-eth/",
```

---

I like `etc-types`